src/theme/index.hbs

```diff
@@ -43,7 +43,11 @@
         <!-- Provide site root to javascript -->
         <script type="text/javascript">
             var path_to_root = "{{ path_to_root }}";
-            var default_theme = window.matchMedia("(prefers-color-scheme: dark)").matches ? "{{ preferred_dark_theme }}" : "{{ default_theme }}";
+            var default_theme = window.matchMedia("(prefers-color-scheme: dark)").matches &&
+                // Fallback to the default theme if preferred_dark_theme isn't available
+                Array.from(document.querySelector('link[href="{{ path_to_root }}css/variables.css"]').sheet.rules)
```

Comment:
@ehuss This is the only way I can find to detect if the theme is available or not. It's *really* janky though (it searches through the stylesheets for a rule affecting the theme). Is there a different way you had in mind/know of?

Reply:
Or, instead of falling back to the light theme from within JS, we could just have a note in the docs to set `preferred-dark-theme` to the same value as `default-theme` to disable it.

> Presumably if someone customizes the themes, they either have a "light" theme, or change the default in the config.

Could that same logic also apply to `preferred-dark-theme`?